package.json

```diff
@@ -60,7 +61,7 @@
     "test-with-coverage": "nyc grunt test",
     "upload-coverage": "cat ./coverage/lcov.info | codacy-coverage",
     "clean-coverage": "rm -rf ./coverage .nyc_cache .nyc_output",
-    "postinstall": "bash copy-vendor.sh"
+    "postinstall": "node shell-scripts/copy-vendor.js"
```

Comment:
Could you use `scripts` directory?

Reply:
```scripts/copy-vendor.js```
or 
```src/copy-vendor.js```

Reply:
I went with my gut instinct on this one.

Reply:
> scripts/copy-vendor.js

Yes, it's fine for me.